pyqt-configure

```diff
@@ -1,18 +1,25 @@
 #!/bin/sh
 
 
-PYTHON=python2
+PYTHON=python3
 PYTHON_VERSION=$(${PYTHON} -c 'import sys; print(sys.version[:3])')
 
-
 ${PYTHON} configure.py \
+    --assume-shared \
     --confirm-license \
     --enable=QtCore \
     --enable=QtGui \
     --enable=QtNetwork \
+    --enable=QtWidgets \
     --no-designer-plugin \
     --no-docstrings \
+    --no-qml-plugin \
     --no-qsci-api \
+    --sysroot=/app \
+    -w \
     --bindir=/app/bin \
     --destdir=/app/lib/python${PYTHON_VERSION}/site-packages \
-    --sipdir=/app/share/sip
+    --sipdir=/app/share/sip \
+    --verbose \
+    QMAKE_CFLAGS_RELEASE="-I/usr/include/python${PYTHON_VERSION}m/" \
+    QMAKE_CXXFLAGS_RELEASE="-I/usr/include/python${PYTHON_VERSION}m/"
```

Comment:
These last 2 don't seem to be needed, I can build fine without them. Am I missing something?